package.json

```diff
@@ -42,16 +42,15 @@
     "@fullhuman/postcss-purgecss": "^2.2.0",
```

Comment:
`@fullhuman/postcss-purgecss`
I think this can be removed too, as it isn't used anywhere in the project.

Reply:
done PTAL now